app/scripts/proactive/view/CatalogPublishView.js

```diff
@@ -313,30 +330,36 @@ define(
                 _(this.buckets.models).each(function(bucket) {
                     var bucketName = bucket.get("name");
                     this.$('#catalog-publish-buckets-table').append(BucketList({bucket: bucket, bucketname: bucketName}));
-                    if (!isWorkflow && bucketName == alreadyPublishedBucketName)
+                    if ( bucketName == alreadyPublishedBucketName ){
                         selectIndex = i;
+                    }
                     i++;
                 }, this);
 
                 if(typeof selectIndex !== "undefined"){
                     // to open the browser on the right bucket
-                    this.internalSelectBucket(this.$('#catalog-publish-buckets-table tr')[selectIndex]);
+                    localStorage.setItem("selectBucket", that.$('#catalog-publish-buckets-table tr')[selectIndex].getAttribute("data-bucketname"));
+                    this.internalSelectBucket(this.$('#catalog-publish-buckets-table tr')[selectIndex], true);
                 } else {
                     // Select the previous bucket if it isn't the first time, otherwise, select the first bucket on the list
                     if(localStorage.selectBucket && that.$('#catalog-publish-buckets-table tr')[0]){
                         const indexOfSelectedBucket = (new Array(that.$('#catalog-publish-buckets-table tr').length)).findIndex(function(elem, index){
                             return that.$('#catalog-publish-buckets-table tr')[index].getAttribute("data-bucketname") == localStorage.selectBucket;
                         })
-                        this.internalSelectBucket(this.$('#catalog-publish-buckets-table tr')[indexOfSelectedBucket > 0 ? indexOfSelectedBucket : 0]);
+                        this.internalSelectBucket(this.$('#catalog-publish-buckets-table tr')[indexOfSelectedBucket > 0 ? indexOfSelectedBucket : 0], true);
                     } else {
                         if(that.$('#catalog-publish-buckets-table tr').length){
                             localStorage.setItem("selectBucket", that.$('#catalog-publish-buckets-table tr')[0].getAttribute("data-bucketname"));
-                            this.internalSelectBucket(this.$('#catalog-publish-buckets-table tr')[0]);
+                            this.internalSelectBucket(this.$('#catalog-publish-buckets-table tr')[0], true);
                         }
                     }
                 }
             }
         },
+        PublishScrollToBucket: function() {
+            var scrollToVal = $('#catalog-publish-modal .catalog-selected-row').position().top - 300;
```

Comment:
How is this minus value 300 calculated? Is it always the same in the future?

Reply:
thanks, I use the parent to calculate the scroll position